src/App.js

```diff
@@ -34,13 +36,19 @@ class PanelButtons extends Component {
   };
 
   onClickNext = () => {
-    if (this.props.panelButtons.next.panel === "save") {
-      this.props.startSaveTrainedModel(this.props.dataToSave);
-    } else if (this.props.panelButtons.next.panel === "continue") {
+    if (
+      this.props.panelButtons.next.panel === "continue" ||
+      this.props.panelButtons.next.panel === "finish"
```

Comment:
Could also use `includes` here...

Reply:
I refactored it with `includes`. Thanks!